dist/money.js

```diff
@@ -1,126 +1,140 @@
-// Generated by CoffeeScript 1.10.0
-(function(root, factory) {
-  if (typeof define === 'function' && define.amd) {
-    define([], factory);
-  } else if (typeof exports === 'object') {
-    module.exports = factory();
+"use strict";
```

Comment:
Do we actually need to include the compiled file in the npm package?

Reply:
Agree, hopefully we can remove this and have npm publish process generate this

Reply:
@mnoack I don't believe you can do that with publish. I believe it only pushes up the current git tags and head. Not modified or untracked files.

Reply:
@jackmcpickle https://docs.npmjs.com/misc/scripts - We have a package.json with prepublish so from my reading it will build it when publishing to NPM.

Can you delete this file and add it to the .gitignore then I think I'm happy to merge this PR

Thanks!